app/forms/keywords_form.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class KeywordsForm
+  include ActiveModel::Model
+
+  validates_with KeywordsFormValidator
+
+  attr_reader :file
+
+  def initialize(user)
+    @user = user
+  end
+
+  def save(file)
+    @file = file
+
+    return false unless valid?
```

Comment:
How about this? I think it easier to read
```suggestion
    return false if invalid?
```

Reply:
Fixed in [`4c447f6` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/4c447f699243eec5c046c00dd38c76964fc91c8f)